ext/ITensorsHDF5Ext/ITensorsHDF5Ext.jl

```diff
@@ -0,0 +1,14 @@
+module ITensorsHDF5Ext
+
+using HDF5: HDF5, attributes, create_group, open_group, read, write
```

Comment:
I realize it is pedantic but I would prefer if this was moved to each file to keep the style consistent with other parts of the code.